acceptance_tests/features/steps/view_collection_exercise_details_ready_for_review_state.py

```diff
@@ -0,0 +1,40 @@
+from behave import given, when, then
+
+from acceptance_tests.features.pages import collection_exercise_details
+
+
+@given('the 201809 collection exercise for the RSI survey is Scheduled')
+def rsi_201809_exists_and_scheduled_displayed(_):
```

Comment:
What actually puts this into scheduled as it's failing for me?

Reply:
The load_event.py script which is being ran in make setup

Reply:
Think I've figured it out. It's a bug when upgrading the database from before https://github.com/ONSdigital/rm-collection-exercise-service changes to after it. It's something I will look into fixing now